doc/syntax/literals.rdoc

```diff
@@ -299,6 +311,11 @@ Like strings, a single-quote may be used to disable interpolation:
 When creating a Hash, there is a special syntax for referencing a Symbol as
 well.
 
+See also:
+
+* {%r: Symbol Literals}[#label-25s-3A+Symbol+Literals]
```

Comment:
This looks wrong. `%r` is for regexp literals, not symbol literals.

Reply:
Fixed.